packages/components/src/components/list-item-location/list-item-location.tsx

```diff
@@ -76,6 +81,8 @@ export class ListItemLocation {
     private iconDisplaySize = parseInt(midtIcon.icon.size.medium.value);
 
     /**
+     * Emits the location to event listeners.
+     *
      * @description Emits the location to event listeners.
```

Comment:
Seems like we are using the `@description` tag throughout this package. Then there is no need for the added like. The result is the same.

Same for other JSDoc blocks.

https://jsdoc.app/tags-description.html : "If there's both a description at the beginning of a JSDoc comment and a description provided with the @description tag, the description specified with the @description will override the description at the beginning of the comment."

Reply:
The code was showing me this warning message almost everywhere where the description was added only with the `@description` tag: "Remove the @description tag to leave a plain block description or add additional description text above the @description line."

Reply:
But you're not adding additional description text. It's the same 😁

Reply:
@ammapspeople I reverted the changes 😁